pullrefresh/src/commonMain/kotlin/dev/materii/pullrefresh/PullRefreshState.kt

```diff
@@ -125,17 +125,23 @@ class PullRefreshState internal constructor(
     internal fun onRelease(velocity: Float): Float {
         if (refreshing) return 0f // Already refreshing, do nothing
 
-        if (adjustedDistancePulled > threshold) {
+        val isRefreshTiming = adjustedDistancePulled > threshold
+        if (isRefreshTiming) {
             onRefreshState.value()
         }
         animateIndicatorTo(0f)
         val consumed = when {
             // We are flinging without having dragged the pull refresh (for example a fling inside
             // a list) - don't consume
             distancePulled == 0f -> 0f
-            // If the velocity is negative, the fling is upwards, and we don't want to prevent the
-            // the list from scrolling
-            velocity < 0f -> 0f
+            velocity < 0f -> if (isRefreshTiming) {
```

Comment:
A one-liner arrow for a multiline if seems weird, use blocks instead
```suggestion
            velocity < 0f -> { 
            if (isRefreshTiming) {
```